### PR TITLE
209 navbar

### PR DESCRIPTION
**Pull request by hsun1316**

### CODE REVIEW

File: public/css/index.less [hunk 1]
```diff
@@ -204,40 +123,45 @@ section {
...
+  margin-right: auto;
+  width: 800px;
+  padding-right: 180px;
+}
+
+.redbox {
+  background-color: #cd5257;
+  //margin-left: calc(~"25% - 150px");
+  //margin-right: calc(~"50% - 120px");
     h2 {
```
Comment (mahsu) [inline public/css/index.less]: can we merge h2, h3,h4, p since they all have the same property?

File: public/css/index.less [hunk 2]
```diff
@@ -204,40 +123,45 @@ section {
...
-    color: @jasonblue;
+    color: @white;
   }
 }
 .big-heading h2 {
-  color: @jasonblue;
-    font-size: 34px;
-    text-transform: lowercase;
-    width:100%;
+  color: @white;
```
Comment (mahsu) [inline public/css/index.less]: fix indentation?

File: public/css/index.less [hunk 3]
```diff
@@ -623,21 +541,16 @@ footer p {
...
-
-  .top-nav-collapse {
-    padding: 0;
-    background-color: #000;
   }
 
-  .navbar-custom.top-nav-collapse {
-    border-bottom: 1px solid rgba(255, 255, 255, .3);
+  .navbar-default .navbar-text {
+    font-weight: 900;
```
Comment (mahsu) [inline public/css/index.less]: use a variable for font weight.

Make sure that `900` is available in the webfont.

File: views/index.jade [hunk 4]
```diff
@@ -1,348 +1,34 @@
...
-            - var msg_class
-            case type.toLowerCase()
-                when "error": -msg_class = "alert-danger"
-                when "success": -msg_class = "alert-success"
-                default: -msg_class = "alert-success"
-            .alert.float.alert-dismissible.float.fadeOut(class=msg_class, role="alert")
-                button.close(type="button", data-dismiss="alert", aria-label="Close"): span(aria-hidden="true") &times;
-                | #{content.toLowerCase()}
-
-    if messages.error
```
Comment (mahsu) [inline views/index.jade]: need to keep 46-49

File: views/index.jade [hunk 5]
```diff
@@ -1,348 +1,34 @@
...
-            .collapse.navbar-collapse.navbar-left.navbar-main-collapse
-                ul.nav.navbar-nav
-                    li: a(href="#about") About
-                    li: a(href="#sustainable") Sustainability
-                    li: a(href="#faq") FAQ
-                    li: a(href="#sponsors") Sponsors
-
-
-    //- Creates a message if content exists, where type:error|success
-    mixin message(content, type)
```
Comment (mahsu) [inline views/index.jade]: need to keep 35-44

Comment (mahsu) [inline views/index.jade]: ideally this should really be refactored to a mixins file, but that's not really a priority.

File: views/index.jade [hunk 6]
```diff
@@ -1,348 +1,34 @@
...
-
-    //- Section: intro
-    section#intro.intro
-        .transparent
-        .intro-wrapper
-            .h1container
-                h1 #[div.icon] BigRed//Hacks
+    nav.navbar.navbar-default
+        .container-fluid
+            .navbar-header
```
Comment (mahsu) [inline views/index.jade]: IMO navbar-header is a bit of a misnomer, sincer they're really navbar items.

File: views/index.jade [hunk 7]
```diff
@@ -1,348 +1,34 @@
...
+                a.navbar-brand(href='#')
+                    img(alt='BigRed//Hacks', src='/img/icon/brh-icon-full.png', width=40, height=40)
+            .navbar-header
+                a.navbar-text(href='google.com') About
```
Comment (mahsu) [inline views/index.jade]: mark this with a todo

File: public/css/index.less [hunk 8]
```diff
@@ -744,7 +647,6 @@ footer p {
     margin-right: 0;
     color: @jasonblue;
```
Comment (mahsu) [inline public/css/index.less]: idk if jasonblue is intentially saved as a fix for later, but feel free to ignore if it is.

File: views/index.jade [hunk 9]
```diff
@@ -1,348 +1,34 @@
...
+                    h2 Data Science Student Challenge
+                  .big-heading2
+                    h2 Powered by Cortana Intelligence Suite
 block footer
-    script(src="/js/vendor/jquery.easing.min.js")
-    script(src="/js/vendor/jquery.scrollTo.js")
-    script(src="//ajax.aspnetcdn.com/ajax/jquery.validate/1.13.1/jquery.validate.min.js")
-    script(src="/js/jquery.validate.options.js")
-    script(src="/js/vendor/skrollr.min.js")
+    script(src="https://code.jquery.com/jquery-3.1.1.min.js")
```
Comment (mahsu) [inline views/index.jade]: change link to `//code.jquery.com/jquery-3.1.1.min.js`. This makes it work with both http and https

File: views/index.jade [hunk 10]
```diff
@@ -1,348 +1,34 @@
...
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/jquery-validate/1.16.0/jquery.validate.min.js")
```
Comment (mahsu) [inline views/index.jade]: Same as above

File: views/index.jade [hunk 11]
```diff
@@ -1,348 +1,34 @@
...
+                .redbox
+                  .big-heading
+                    h2 Data Science Student Challenge
+                  .big-heading2
+                    h2 Powered by Cortana Intelligence Suite
 block footer
-    script(src="/js/vendor/jquery.easing.min.js")
-    script(src="/js/vendor/jquery.scrollTo.js")
-    script(src="//ajax.aspnetcdn.com/ajax/jquery.validate/1.13.1/jquery.validate.min.js")
-    script(src="/js/jquery.validate.options.js")
```
Comment (mahsu) [inline views/index.jade]: Also if you're going to include validate, also keep this.

File: views/index.jade (hunk 10 above)
Comment (mahsu) [inline views/index.jade]: What are the risks for upgrading from 1.13.1 to 1.16.0, if any?

**Comment by julianahhong** [comment]
@hsun1316 can you add the white potion image to the redish box?

File: public/css/index.less (hunk 3 above)
Comment (julianahhong) [inline public/css/index.less]: @hsun1316 I think you still have to address this comment? you can make a variable for font-weight in the _global.less file and reference it here in the index.less file

**Pull request merged**